common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3770,6 +3774,27 @@ object CelebornConf extends Logging {
       .checkValues(Set(ShuffleMode.HASH.name, ShuffleMode.SORT.name))
       .createWithDefault(ShuffleMode.HASH.name)
 
+  val CLIENT_PUSH_DYNAMIC_WRITE_MODE_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.spark.push.dynamicWriteMode.enabled")
+      .categories("client")
+      .doc("Whether to dynamically switch push write mode based on conditions.If true, " +
+        s"shuffle mode will be is determined by partition count," +
+        s"not ${SPARK_SHUFFLE_WRITER_MODE.key}")
```

Comment:
it's better to see mention it the `SPARK_SHUFFLE_WRITER_MODE`'s docs, for example

This configuration only takes effect when `${CLIENT_PUSH_DYNAMIC_WRITE_MODE_ENABLED.key}` is false

Reply:
Thanks for you suggestion,Updated